jsx/DataTable.js

```diff
@@ -497,6 +497,14 @@ class DataTable extends Component {
       csvData = filteredData;
     }
 
+    // Map cell data to proper values if applicable.
+    if (this.props.getMappedCell) {
+      csvData = csvData
+      .map((row, i) => this.props.fields
+        .map((field, j) => this.props.getMappedCell(field.label, row[j]));
+      );
+    }
```

Comment:
1. Is this going to conflict with #5299? If so, that PR is probably more important right now and I think this should be blocked until it goes in.
2. Can you update some existing modules in this PR?

Reply:
1. No, it shouldn't conflict! I think these PRs can be made separately. 
2. Sure can!